gallery/src/styles/NxCard/NxCardVerticalExample.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardVerticalExample() {
+  return (
+    <>
+      <div className="nx-card-container nx-card-container--row">
+        <div className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <span className="nx-card__call-out-text">42</span>
+          </div>
+          <div className="nx-card__text">
+            Text
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </div>
+        <div className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} />
+          </div>
+          <div className="nx-card__text">
+            Data point details
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </div>
+        <div className="nx-card">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} className="nx-icon--large" />
+          </div>
+          <div className="nx-card__text">
+            Large icon
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </div>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
```

Comment:
I see you've got one container per row.  I was expecting a single container which would use a multi-line flexbox to arrange the cards into potentially multiple rows.  That would be significantly more helpful I think as it would allow the caller to throw any number of cards into a single container and have them lay out right.  With the container-per-row approach, the caller would have to know how many cards to fit into each row.  Which would be especially challenging give how unpredictable their widths are.

Reply:
> I see you've got one container per row. I was expecting a single container which would use a multi-line flexbox to arrange the cards into potentially multiple rows. That would be significantly more helpful I think as it would allow the caller to throw any number of cards into a single container and have them lay out right. With the container-per-row approach, the caller would have to know how many cards to fit into each row. Which would be especially challenging given how unpredictable their widths are.

Yeah that makes sense, though it has the interesting side-effect that the cards no longer respect `container-horizontal` which means the example wraps an extra line. https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/b3baf7ff7290bce1bd0a3c1e0b7bea044822f58e

FYI I left the three minimal examples in their own rows so that they weren't stretched vertically.

Reply:
> Yeah that makes sense, though it has the interesting side-effect that the cards no longer respect container-horizontal which means the example wraps an extra line. b3baf7f

I'm not sure what was happening before, but from what I can tell now that's because `.nx-card-container--row` simply doesn't include `container-horizontal`.

Reply:
> > Yeah that makes sense, though it has the interesting side-effect that the cards no longer respect container-horizontal which means the example wraps an extra line. [b3baf7f](https://github.com/sonatype/sonatype-react-shared-components/commit/b3baf7ff7290bce1bd0a3c1e0b7bea044822f58e)
> 
> I'm not sure what was happening before, but from what I can tell now that's because `.nx-card-container--row` simply doesn't include `container-horizontal`.

I removed it because it wasn't doing anything useful. As far as the browser (or SCSS) is concerned in the case where we wrap the flex elements the element that corresponds to `:last-child` is the last instance of `nx-card` not the last card in any row. So in the case where we have 9 cards that have wrapped into 3 rows (as in the example), the 3rd card still has it's right margin even though it is the last card in its row, the 5th card still has it's margin even though it is the last card in its row, and the last card has its right margin removed because it is the `:last-child`. 

<img width="420" alt="Screen Shot 2020-10-16 at 11 04 31 AM" src="https://user-images.githubusercontent.com/1991443/96275178-6e51bf80-0f9f-11eb-9548-677beee65abf.png">

`column-gap` does not solve this. Regardless I don't think that this is worth worrying about with our max-width setting as it is three cards at max width cannot fit in one row.